lib/galaxy_test/api/test_workflows.py

```diff
@@ -1352,6 +1367,22 @@ def test_workflow_invocation_report_custom(self):
             assert "## Workflow Inputs" in markdown_content
             assert "## About This Report" in markdown_content
 
+    @skip_without_tool("cat1")
+    def test_export_invocation_bco(self):
+        with self.dataset_populator.test_history() as history_id:
+            summary = self._run_jobs(WORKFLOW_SIMPLE, test_data={"input1": "hello world"}, history_id=history_id)
+            invocation_id = summary.invocation_id
+            bco = self._get("invocations/%s/export_bco" % invocation_id).json()
+            self._assert_has_keys(bco, "bco_id", "spec_version", "etag", "provenance_domain", "usability_domain", "description_domain", "execution_domain", "parametric_domain", "io_domain", "error_domain")
+            # self.assertEqual(bco['spec_version'], 'https://w3id.org/biocompute/1.4.0/')
```

Comment:
I think you should be able to restore the assert now:
```suggestion
            self.assertEqual(bco['spec_version'], 'https://w3id.org/biocompute/1.4.0/')
```